packages/network/lib/cors.ts

```diff
@@ -11,6 +11,8 @@ const debug = debugModule('cypress:network:cors')
 // match IP addresses or anything following the last .
 const customTldsRe = /(^[\d\.]+$|\.[^\.]+$)/
 
+const strictSameOriginDomains = Object.freeze(['google.com'])
```

Comment:
@mjhenkes can we allow somehow parametrizing this? we have mobile tests that are on the same domain, but they require different user-agent. we were trying to override it with `cy.origin()` and it worked perfectly well till this PR change...

Reply:
@rnovosad would you be able to share an example?

Reply:
@AtofStryker I provided it in the comment https://github.com/cypress-io/cypress/issues/2100#issuecomment-1323948786

Reply:
@rnovosad got it. I will reply to the thread there!